### ISSUE TITLE
[Gentoo] Unable to locate installer for your Linux...

### ISSUE THREAD

**Issue by pauledd** [beefproject/beef#1742]
#### Environment

using latest git master

ruby 2.4.6p354

None

Gentoo Linux

#### Configuration

Are you using a non-default configuration?
No.

Have you enabled or disabled any BeEF extensions?
No.

#### Summary

Cannot install beef

#### Expected Behaviour

Can install beef

#### Actual Behaviour

```
[INFO]  Detecting OS...
[INFO]  Operating System: Linux
[INFO]  Launching Linux install...
[INFO]  Detecting Linux OS distribution...
[FATAL]  Unable to locate installer for your Linux distribution
```
#### Steps to Reproduce
mkdir build && cd build
git clone https://github.com/beefproject/beef.git
cd beef
./install

Can someone please give me a hint what I am doing wrong?

**Comment by pauledd** [comment]
solved by manually issuing:
`bundle install --without test development`

**Comment by ridhamsharma12345** [comment]
Bro is also have the same problem

**Comment by yhackerKennY** [comment]
Me too

**Comment by bcoles** [comment]
The installer script is targeted towards popular Linux distributions and meant for lazy people. There's no installer for Gentoo.

The requirements are described here:

* https://github.com/beefproject/beef#requirements

And further details are provided in the installation documentation:

* https://github.com/beefproject/beef/blob/master/INSTALL.txt
* https://github.com/beefproject/beef/wiki/Installation

In short: install Ruby, install Ruby Gems, `gem install bundler`, `bundle install`. You'll also need Nodejs. You'll probably also need some sqlite libs.
